packages/pieces/community/krisp-call/src/lib/triggers/index.ts

```diff
@@ -0,0 +1,80 @@
+import { krispcallAuth } from '../../index';
+import { HttpMethod, httpClient } from "@activepieces/pieces-common";
+import { TriggerStrategy, createTrigger } from "@activepieces/pieces-framework"
+
+export const triggers = [
+    {
+        name: 'newVoicemail',
+        displayName: 'New Voicemail',
+        description: 'Trigger when a new voicemail is received.',
+        action: 'new_voicemail',
+        sampleData: {
+            id: '',
+            from: '+9779821110987',
+            duration: '5 seconds',
+            call_time: '2000-10-31T01:30:00.000-05:00',
+            voicemail_audio: 'voicemail.mp4',
+        },
+    },
+    {
+        name: 'newMms',
+        displayName: 'New MMS',
+        description: 'Trigger when a new MMS is received.',
+        action: 'new_sms_or_mms',
+        sampleData: {},
```

Comment:
Hi @deependra321 

I have refactored to be one file as they are similars, can you add sample data for New MMS?

And we are good to go

Reply:
{
        "id": "YiW2nyxqtJPYqkRKbrcJQ7",
        "from_number": "+16466813538",
        "to_number": "+12517327005",
        "content": "Last testing",
        "media_link": "https://api.twilio.com/2010-04-01/Accounts/ACf588e133cdf37aca232fa7527e0f6080/Messages/MM0192dba31f26ac91469056de3cb126a7/Media/MEe56cb6ce56db15ab3473cdb85319bb41"
    },

Reply:
sample data is above for new sms/mms can you add it.